caos.ansible_roles/roles/install_crowdstrike_falcon/tasks/win.yaml

```diff
@@ -11,29 +11,24 @@
       Accept: "application/json"
       Content-Type: "application/x-www-form-urlencoded"
   register: crowdstrike_auth
-  when: ansible_facts['os_family'] == 'Windows'
-  
+
 - name: 'Download CrowdStrike Falcon EXE'
   win_get_url:
-    url: "{{ api_base_url }}/sensors/entities/download-installer/v1?id={{ item.hash }}"
+    url: "{{ api_base_url }}/sensors/entities/download-installer/v1?id={{ win[0].hash }}"
     url_timeout: 120
     headers:
       Authorization: "Bearer {{ crowdstrike_auth.json.access_token }}"
       Accept: "application/octet-stream"
       Content-Type: "application/json"
-    dest: C:\Users\ansible\Downloads\falcon-sensor.exe
-  when: ansible_facts['os_family'] == 'Windows' and inventory_hostname is search(item.hostname)
-  loop: "{{ win }}"
```

Comment:
I'd probably revert this and still allow it to iterate through a list of supported packages